lib/linguist/languages.yml

```diff
@@ -44,6 +44,16 @@ AGS Script:
   tm_scope: source.c++
   ace_mode: c_cpp
 
+Alex Haskell:
+  type: programming
+  group: Haskell
```

Comment:
We use a Bayesian classifier (and heuristics only in some special cases). So if you open a pull request to add `.x` to Haskell with a sample files (to train the classifier) it should be okay :)

If there are too many conflicts with Logos, we'll see if we can define a heuristic.